pandas/tests/frame/test_sorting.py

```diff
@@ -21,75 +21,63 @@ class TestDataFrameSorting(tm.TestCase, TestData):
 
     _multiprocess_can_split_ = True
 
-    def test_sort_values(self):
-        # API for 9816
-
-        # sort_index
+    def test_sort_index(self):
         frame = DataFrame(np.arange(16).reshape(4, 4), index=[1, 2, 3, 4],
                           columns=['A', 'B', 'C', 'D'])
 
-        # 9816 deprecated
-        with tm.assert_produces_warning(FutureWarning):
-            frame.sort(columns='A')
-        with tm.assert_produces_warning(FutureWarning):
-            frame.sort()
-
+        # axis=0 : sort rows by index labels
         unordered = frame.ix[[3, 2, 4, 1]]
-        expected = unordered.sort_index()
-
         result = unordered.sort_index(axis=0)
+        expected = frame
         assert_frame_equal(result, expected)
 
-        unordered = frame.ix[:, [2, 1, 3, 0]]
-        expected = unordered.sort_index(axis=1)
+        result = unordered.sort_index(ascending=False)
+        expected = frame[::-1]
+        assert_frame_equal(result, expected)
 
+        # axis=1 : sort columns by column names
+        unordered = frame.ix[:, [2, 1, 3, 0]]
         result = unordered.sort_index(axis=1)
-        assert_frame_equal(result, expected)
+        assert_frame_equal(result, frame)
+
+        result = unordered.sort_index(axis=1, ascending=False)
+        expected = frame.ix[:, ::-1]
         assert_frame_equal(result, expected)
 
-        # sortlevel
-        mi = MultiIndex.from_tuples([[1, 1, 3], [1, 1, 1]], names=list('ABC'))
+        # sort rows by specified level of multi-index
```

Comment:
I would make multi-index part of a 2nd test (e.g. separate long tests into multiple ones)